tracex_project/tracex/templates/landing_page.html

```diff
@@ -10,26 +10,26 @@
     <div class="landing_page_container">
         <img src="{% static '/tracex/img/TraceEx_v2.svg' %}" alt="TracEX Logo" class="landing_page_logo">
         <div class="landing_page_menu">
-            <div class="item patient_journey_generator">
-                <a href="{% url 'journey_generator_overview' %}">
-                <img src="{% static '/tracex/img/patient_journey_generator_logo.png' %}" alt="Patient Journey Generator Logo" class="patient_journey_generator_logo">
-                <span class="item_text">Patient Journey Generator</span>
+            <div class="item patient_journey_generator" style="display: flex;">
```

Comment:
I would suggest to outsource the css in the style.css, especially because its the same for all 4 tiles